sequence.go

```diff
@@ -92,3 +92,21 @@ func getFeatureSequence(feature Feature, location Location) string {
 
 	return sequenceString
 }
+
+//AnonymizeSequence takes a sequence struct and substitute identification data for a new anonymized string and returns this modified Sequence struct.
+func anonymizeSequence(sequence Sequence, sequence_id string) Sequence {
+
+        anonymizer := "anonymized"
```

Comment:
this parts actually unneeded ID is generated via hashing and is directly related to sequence string itself anyways.